presets/jscs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "preset": "google",
```

Comment:
What if the rule in configuration files was extendPreset instead of preset? Just like you can extend by path. Or in some other way merge those two options.